doc/manual/metaprogramming.rst

```diff
@@ -118,8 +137,22 @@ instead of an :obj:`Expr`:
     julia> typeof(ans)
     Symbol
 
+:obj:`Symbol`\ s can also be created using :func:`symbol`, which takes
+a character or string as its argument:
+
+.. doctest::
+
+    julia> :foo == symbol("foo")
+    true
+
+    julia> symbol('\'')
```

Comment:
I'm not sure I was clear. I think introducing `symbol(::Char)` via `symbol('\'')` is not the clearest solution, as the reader may be confused by simultaneous introduction of two different meanings of `'`. Better start with `symbol("'")` first, since it's more similar to `symbol("foo")` which was introduced above; or replace `symbol('\'')` with `symbol('&')`. (Escaping with `\'` is not essential for metaprogramming, and this section is already dense enough.)